src/main/java/org/toradocu/translator/ConditionTranslator.java

```diff
@@ -630,6 +630,64 @@ private static String translateSecondPart(String text, DocumentedMethod method)
       translatePropositions(propositions, method);
```

Comment:
Can we merge this code with the following to avoid to call the translation phase multiple times? (`translatePropositions` method is also called in a later part of the code.)